.github/workflows/publish-rust-find-changes.sh

```diff
@@ -0,0 +1,26 @@
+#/bin/bash
+
+image_name="apollo-router"
+github_org="graphql-hive"
+router_version=$(cargo tree -i apollo-router --quiet | head -n 1 | awk -F" v" '{print $2}')
+plugin_version=$(jq -r '.version' packages/libraries/router/package.json)
+has_changes=$(git diff HEAD~ HEAD --name-only -- 'packages/libraries/router/Cargo.toml' 'Cargo.lock' 'packages/libraries/router/package.json')
+
+if [ "$has_changes" ]; then
+  image_tag_version="router${router_version}-plugin${plugin_version}"
+
+  response=$(curl -L \
+    -H "Accept: application/vnd.github+json" \
+    -H "Authorization: Bearer ${GITHUB_TOKEN}" \
```

Comment:
will this be erased by Github or will we see it in logs?

Reply:
Redacted 
<img width="901" alt="image" src="https://github.com/user-attachments/assets/c8eee79d-362c-4a47-97e7-6a96f7f70e2e">